src/Skoruba.IdentityServer4.Admin.BusinessLogic/Services/ClientService.cs

```diff
@@ -66,6 +66,9 @@ private void PrepareClientTypeForNewClient(ClientDto client)
                 case ClientType.Machine:
                     client.AllowedGrantTypes.AddRange(GrantTypes.ResourceOwnerPasswordAndClientCredentials);
                     break;
+                case ClientType.Device:
+                    client.AllowedGrantTypes.AddRange(GrantTypes.DeviceFlow);
```

Comment:
Maybe add here:
```
client.RequireClientSecret = false;
```

since in most scenarios this client won't require client secret?

Reply:
Good point, I've added.

Reply:
Maybe we could add default allow of offline_access while we're at it?

Reply:
Done. :)